src/main/scala/net/cakesolutions/CakeDockerComposePlugin.scala

```diff
@@ -50,7 +50,11 @@ object CakeDockerComposePlugin extends AutoPlugin {
         yaml => Seq("-f", yaml.getCanonicalPath)
       )
     val result =
-      Process(Seq("docker-compose") ++ projectOverrides ++ Seq("down")).!
+      Process(
+        Seq("docker-compose") ++
+          projectOverrides ++
+          Seq("down", "--rmi", "all", "--volumes", "--remove-orphans")
```

Comment:
I think this was a mistake. We should have a `dockerRemove` for this. This significantly slows down dev turnaround by deleting large images downloaded from the interwebz.